types/gsap/index.d.ts

```diff
@@ -1,6 +1,10 @@
-// Type definitions for GSAP 1.19
+// Type definitions for GSAP 1.20.6
 // Project: http://greensock.com/
-// Definitions by: VILIC VANE <https://vilic.github.io/>, Robert S <https://github.com/codebelt>, Richard Fox <https://github.com/ProbablePrime>, Philip Bulley <https://github.com/philipbulley>
+// Definitions by:  VILIC VANE <https://vilic.github.io/>,
+//                  Robert S <https://github.com/codebelt>,
```

Comment:
Is this format ok?

The bot uses these to tag reviewers... LIKE ME :)

Reply:
I followed what is being shown in [the readme](https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package).

Reply:
Ahh ok